src/app/pages/sign-up/sign-up.component.html

```diff
@@ -1 +1,16 @@
-<p>sign-up works!</p>
+<div class="step-header">
+  <div class="step-header__step step-header__step_active"></div>
+  <div
+    class="step-header__step"
+    [class.step-header__step_active]="step >= 2"
+  ></div>
+</div>
+
+<header class="header">
+  <p class="step">Step {{ step }} of 2</p>
+  <p class="title">Sign up</p>
+  <p>Already have account? <a class="link" routerLink="/log-in">Log in!</a></p>
+</header>
+
+<app-sign-up-first-step *ngIf="step === 1"></app-sign-up-first-step>
+<app-sign-up-second-step *ngIf="step === 2"></app-sign-up-second-step>
```

Comment:
can be done with nested routing, more stable solution

Reply:
@dmitry-darly-solutions 
Fixed